apps/gp2-server/test/routes/working-group.route.test.ts

```diff
@@ -0,0 +1,50 @@
+import supertest from 'supertest';
+import { appFactory } from '../../src/app';
+import { getListWorkingGroupResponse } from '../fixtures/working-group.fixtures';
+import { authHandlerMock } from '../mocks/auth-handler.mock';
+import { workingGroupControllerMock } from '../mocks/working-group-controller.mock';
+
+describe('/working-groups/ route', () => {
+  const app = appFactory({
+    workingGroupController: workingGroupControllerMock,
+    authHandler: authHandlerMock,
+  });
+
+  afterEach(() => {
+    jest.clearAllMocks();
```

Comment:
same as the resetAllMocks comment above.

Reply:
in the one above it was used twice so i removed it - here its only applied once, and we do need it